frontend/src/components/editor/chrome/types.ts

```diff
@@ -23,23 +24,18 @@ import { isWasm } from "@/core/wasm/utils";
 
 export type PanelType =
   | "files"
-  | "errors"
   | "variables"
   | "outline"
   | "dependencies"
-  | "tracing"
   | "packages"
   | "documentation"
   | "snippets"
   | "datasources"
```

Comment:
Thoughts on this going in dev tools? Was just thinking we could offer a "inspect element" like behavior with clicking on a cell highlights it in the graph. We have a "jump to current cell", but it doesn't have a toggle.

Just a suggestion

Reply:
In a follow up PR, i consolidated variables and datasources into the same panel #7459. I also like @mscolnick bigger vision of letting folks adjust which panel items are in sidebar vs developer panel (like in VS Code). We just need to have more responsive components so I'd like to have that for a follow up PR.